.github/workflows/release.yml

```diff
@@ -0,0 +1,152 @@
+name: Create Draft Release
+
+on:
+  push:
+    paths:
+    - 'CHANGELOG'
+  workflow_dispatch:
+
+
+jobs:
+  enumerate:
+    runs-on: ubuntu-latest
+    outputs:
+      runtime: ${{ steps.generate-matrix.outputs.runtime }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Generate runtime matrix
+        id: generate-matrix
+        run: |
+          >>$GITHUB_OUTPUT echo runtime="[$(find * -name "lib.rs" -not -path "*/common/*" -not -path "*/constants/*" | awk -F/ '{sub("/src/lib.rs",""); print "[\"" gensub("/" $NF "$", "", "g") "\",\"" $NF "\"],"}' | tr -d $'\n')]"
+  build-runtimes:
+    continue-on-error: true
+    runs-on: ubuntu-latest
+    needs:
+      - enumerate
+    strategy:
+      matrix:
+        runtime: ${{ fromJSON(needs.enumerate.outputs.runtime) }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+
+      - name: Cache target dir
+        uses: actions/cache@v3
+        with:
+          path: "${{ github.workspace }}/${{ matrix.runtime[0] }}/${{ matrix.runtime[1] }}/target"
+          key: srtool-target-${{ matrix.runtime[0] }}-${{ matrix.runtime[1] }}-${{ github.sha }}
+          restore-keys: |
+            srtool-target-${{ matrix.runtime[0] }}-${{ matrix.runtime[1] }}-
+            srtool-target-${{ matrix.runtime[0] }}-
+
+      - name: Build ${{ matrix.runtime[1] }}
+        id: srtool_build
+        uses: chevdor/srtool-actions@v0.7.0
+        with:
+          chain: ${{ matrix.runtime[1] }}
```

Comment:
I think this is currently building with `--release` instead or `--production`.  
Is it possible to overwrite the profile here? Otherwise we could change the default in srtool.

Reply:
FYI I can't find anywhere where we're setting `WASM_BUILD_TYPE`. @chevdor and I think that the runtimes are currently built with the `--release` flag.

Reply:
`srtool` (the image) uses whatever is passed with the [PROFILE](https://github.com/paritytech/srtool/blob/master/scripts/build#L79) ENV or defaults to `release` indeed.

`srtool-actions` (the GHA) does not support passing the profile [yet](https://github.com/chevdor/srtool-actions/issues/18). As a result, `srtool`'s default will be used.

@gilescope started working on a [PR](https://github.com/chevdor/srtool-actions/pull/21) but that will have no effect if the wasm-builder does not support it.

Reply:
We should really fix this, since all the benchmarks are dont with `production` profile.  
This means that the weights for Polkadot/Kusama are currently all wrong.

Reply:
We also need to enable the following feature: https://github.com/paritytech/polkadot/blob/1a50cb1aede201e08df18d9deb501e32f833f40b/runtime/polkadot/Cargo.toml#L300

@coderobe could you please use `--profile production` and `--features on-chain-release-build`. We should pass this here explicitly to be sure.

Reply:
@bkchr It now builds with the production profile. I don't know whether it's possible to configure that feature flag with srtool-actions, however.

Reply:
Srtool should already use it by default https://github.com/paritytech/srtool/blob/a72f77ff200b3754034abb5aff9e2a25ea1c0f31/scripts/getBuildOpts.sh#L23